tests/execution/test-batch-operator-post.md

```diff
@@ -1,6 +1,8 @@
 # test-batch-operator-post
 
-###### sdl error
+---
+expect_validation_error: true
+---
```

Comment:
```suggestion
---
expect_validation_error: true
---
```
@coderabbitai write a script to move all of this code to the top of the file. Moving everything else below it.